dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -18,23 +18,25 @@ struct Clip: Hashable {
     let thumbnail: UIImage
     let isHighlighted: Bool
     let metadata: ClipMetadata
+    let showCode: Bool
     
-    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata) {
+    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata, showCode: Bool) {
         self.url = url
         self.thumbnail = thumbnail
         self.isHighlighted = isHighlighted
         self.metadata = metadata
+        self.showCode = showCode
     }
 }
 
 class ClipsViewModel: ObservableObject, ImagePickerMessenger {
     @Published var clips: [Clip] = [
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=QH2-TGUlwu4") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: true),
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=9bZkp7q19f0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false),
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=p3G5IXn0K7A") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false),
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=sXWjwUl949Y") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false),
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=h7MYJghRWt0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false),
-//            Clip(url: URL(string: "https://www.youtube.com/watch?v=njos57IJf-0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=QH2-TGUlwu4") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: true, showCode: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=9bZkp7q19f0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false, showCode: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=p3G5IXn0K7A") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false, showCode: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=sXWjwUl949Y") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false, showCode: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=h7MYJghRWt0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false, showCode: false),
+//            Clip(url: URL(string: "https://www.youtube.com/watch?v=njos57IJf-0") ?? URL(fileURLWithPath: ""), thumbnail: UIImage(), isHighlighted: false, showCode: false),
```

Comment:
The changes to the initialization of `Clip` instances are also a good improvement, as they now include the `showCode` parameter to control the display of code. This ensures consistency and maintainability of the code.